include/swift/Parse/Parser.h

```diff
@@ -1755,6 +1755,7 @@ class Parser {
   /// expand availability macros when /p ParsingMacroDefinition is false.
   ParserStatus
   parseAvailabilitySpecList(SmallVectorImpl<AvailabilitySpec *> &Specs,
+                            bool isUnavailability = false,
```

Comment:
Nit: This function has 3 call-sites; I think it's best to avoid defaulting this argument, so that call-sites are clearer and can be marked with `/*isUnavailability*/value`.

Reply:
Nevermind, my later comment makes this redundant.